### DISCUSSION TITLE
Consumer subtask 0 received confirmation for unknown checkpoint id 4

### DISCUSSION THREAD

**Discussion by weitianpei** [apache/flink-cdc#754] [Q&A]
WARN  com.alibaba.ververica.cdc.debezium.DebeziumSourceFunction  - Consumer subtask 0 received confirmation for unknown checkpoint id 4

It is my log above. Anyone who can help me?

**Comment by Sukang1002** [comment]
I also threw this error when using MongoDB cdc, reading from an instance.  有大哥帮忙看一下嘛

**Reply by yuxiqian** [reply]
Hi @Sukang1002, Could you please provide more error message & context about your problem? If you believe it's a bug, please create a ticket on [Flink Jira](https://issues.apache.org/jira/projects/FLINK/summary). Thanks!

**Reply by Sukang1002** [reply]
抱歉，表述错误，没有报错，这是单纯的warning，现象是source无数据流入，我写了一个sql，读MongoDB 到 print，唯一有怀疑的点的是MongoDB是从实例
WARN  com.ververica.cdc.debezium.DebeziumSourceFunction    - Consumer subtask 0 received confirmation for unknown checkpoint id 1

**Reply by yuxiqian** [reply]
Can you visit Flink Web UI, select your submitted job, check the metric panel and see if there's any record flow in the graph?

(Also, if it's suspected to be a Debezium related issue, you may use `'scan.incremental.snapshot.enabled' = 'true'`, and Debezium backend will not be used then.)

**Reply by Sukang1002** [reply]
好的，谢谢，我尝试加一下对应的参数看看

**Reply by Sukang1002** [reply]
已解决了，读的admin库导致的，后续将新建业务库来解决这种情况

**Reply by zhuxt2015** [reply]
> 已解决了，读的admin库导致的，后续将新建业务库来解决这种情况

我这边也遇到这种问题，oracle读取不到数据，请问新建一个其他库就解决了吗？
